### PR TITLE
update navigationMenu hook example to show use of 'attr' parameter

### PR DESCRIPTION
**Pull request by monishdeb**
Ref https://github.com/civicrm/civicrm-core/pull/15861

### CODE REVIEW

File: docs/hooks/hook_civicrm_navigationMenu.md [hunk 1]
```diff
@@ -134,7 +134,8 @@ function donortrends_civicrm_navigationMenu(&$params) {
       'separator' => 1,
       'parentID' => $reportID,
       'navID' => $navId,
-      'active' => 1
+      'active' => 1,
+      'attr' => ['target' => '_blank'],   // if you want to open this menu link in a new page, then add '_target' attribute
```
Comment (maxtsero) [inline docs/hooks/hook_civicrm_navigationMenu.md]:
```suggestion
      'attr' => ['target' => '_blank'],
```

We try and avoid code comments - can you remove the code comment and document the "attr" parameter at the top of the file in the "Parameters" section.

**Comment by MikeyMJCO** [comment]
Sorry - logged into the wrong account there. Redoing review

File: docs/hooks/hook_civicrm_navigationMenu.md (hunk 1 above)
Comment (MikeyMJCO) [inline docs/hooks/hook_civicrm_navigationMenu.md]:
```suggestion
      'attr' => ['target' => '_blank'],
```
We try and avoid code comments to document implementation. Can you document the "attr" parameter in the Parameters section of the document?

**Comment by MikeyMJCO** [comment]
@monishdeb can you address the feedback on this please?

**Comment by MikeyMJCO** [comment]
Merging this without the code comment.

**Pull request merged**